test/scan.cpp

```diff
@@ -262,3 +267,18 @@ TEST(Accum, DocSnippet) {
     array gold_accumB_dim1(3, 3, h_gold_accumB_dim1);
     ASSERT_ARRAYS_EQ(gold_accumB_dim1, accumB_dim1);
 }
+
+TEST(Scan, ExclusiveSum1D) {
+    const int block_work_size = 6912;
```

Comment:
Can you make this 20000? I understand why you used 6912 but its an implementation detail and it may change in the future.

Reply:
Yes, and we want 20000 to be the size of the whole input array, right? Currently, the input size this number * 4.